src/core/modules/SolarSystem.cpp

```diff
@@ -3287,8 +3343,13 @@ bool SolarSystem::getFlagPermanentOrbits() const
 
 void SolarSystem::setOrbitsThickness(int v)
 {
-	Planet::orbitsThickness=v;
-	emit orbitsThicknessChanged(v);
+	if (v!=Planet::orbitsThickness)
+	{
+		Planet::orbitsThickness=v;
+		if (StelApp::getInstance().getFlagImmediateSave())
+			StelApp::getInstance().getSettings()->setValue("Int", v);
```

Comment:
Wrong parameter?

Reply:
Of course. Not sure where this typo comes from.